deployer/run.sh

```diff
@@ -0,0 +1,375 @@
+    # use files from secret if present
+    cp /secret/hawkular-metrics.pem $dir
+    cp /secret/hawkular-metrics-ca.cert $dir
+else #fallback to creating one
+    openshift admin ca create-server-cert  \
+      --key=$dir/hawkular-metrics.key \
+      --cert=$dir/hawkular-metrics.crt \
+      --hostnames=hawkular-metrics,${hawkular_metrics_hostname} \
+      --signer-cert="$dir/ca.crt" --signer-key="$dir/ca.key" --signer-serial="$dir/ca.serial.txt"
+    cat $dir/hawkular-metrics.key $dir/hawkular-metrics.crt > $dir/hawkular-metrics.pem
+    cp $dir/ca.crt $dir/hawkular-metrics-ca.cert  
+fi
+
+# Use existing or generate new Hawkular Cassandra certificates
+if [ -n "${HAWKUKAR_CASSANDRA_PEM}" ]; then
+    echo "${HAWKULAR_CASSANDRA_PEM}" | base64 -d > $dir/hawkular-cassandra.pem
+elif [ -s /secret/hawkular-cassandra.pem ]; then
+    # use files from secret if present
+    cp /secret/hawkular-cassandra.pem $dir
+    cp /secret/hawkualr-cassandra-ca.cert $dir
+else #fallback to creating one
+    openshift admin ca create-server-cert  \
+      --key=$dir/hawkular-cassandra.key \
+      --cert=$dir/hawkular-cassandra.crt \
+      --hostnames=hawkular-cassandra \
+      --signer-cert="$dir/ca.crt" --signer-key="$dir/ca.key" --signer-serial="$dir/ca.serial.txt"
+    cat $dir/hawkular-cassandra.key $dir/hawkular-cassandra.crt > $dir/hawkular-cassandra.pem
+    cp $dir/ca.crt $dir/hawkular-cassandra-ca.cert
+fi
+
+# Use existing or generate new Heapster certificates
+if [ -n "${HEAPSTER_CERT}" ]; then
+  echo "${HEAPSTER_CERT}" | base64 -d > $dir/heapster.cert
+  echo "${HEAPSTER_KEY}" | base64 -d > $dir/heapster.key
+elif  [ -s /secret/heapster.cert ]; then
+    # use files from secret if present
+    cp /secret/heapster.cert $dir
+    cp /secret/heapster.key $dir
+else #fallback to creating one
+    openshift admin ca create-server-cert  \
+      --key=$dir/heapster.key \
+      --cert=$dir/heapster.cert \
+      --hostnames=heapster \
+      --signer-cert="$dir/ca.crt" --signer-key="$dir/ca.key" --signer-serial="$dir/ca.serial.txt"
+fi
+
+# Get the Heapster allowed users
+if [ -n "${HEAPSTER_ALLOWED_USERS}" ]; then
+  echo "${HEAPSTER_ALLOWED_USERS}" | base64 -d > $dir/heapster_allowed_users
+elif [ -s /secret/heapster_allowed_users ]; then
+  cp /secret/heapster_allowed_users $dir
+else #create an empty allowed users
+  echo "" > $dir/heapster_allowed_users
+fi
+
+# Get the Heapster Client CA
+if [ -n "${HEAPSTER_CLIENT_CA}" ]; then
+  echo "${HEAPSTER_CLIENT_CA}" | base64 -d > $dir/heapster_client_ca.cert
+elif [ -s /secret/heapster_client_ca.cert ]; then
+  cp /secret/heapster_client_ca.cert $dir
+else #use the ca we already have for signing our own certificates
+  cp $dir/ca.crt $dir/heapster_client_ca.cert
+fi
+  
+
+echo 03 > $dir/ca.serial.txt  # otherwise openssl chokes on the file
+
+# Convert the *.pem files into java keystores
+echo "Generating randomized passwords for the Hawkular Metrics and Cassandra keystores and truststores"
```

Comment:
Is there a need to echo these passwords to whom ever is running this?  do they have need of them in future?